code/game/machinery/computer/crew.dm

```diff
@@ -137,16 +137,27 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 
 	var/list/results = list()
 
-	for(var/mob/living/carbon/human/H as() in GLOB.suit_sensors_list)
-		if(!H)
+	for(var/tracked_mob in GLOB.suit_sensors_list)
```

Comment:
Our sensors are constructed differently and should allow for only carbon/human to be inside. You can do typed as() loop

Reply:
TGs as well, the check for human is later with a stack_trace so you can figure out what happened to somehow get into the sensors list

Reply:
I guess it would remove 2 variable assignments later. As a sidenote, letting carbons be on the suit_sensors_list is what made it shit itself so many times, that's why the human check